pages/common/xprop.md

```diff
@@ -0,0 +1,20 @@
+# xprop
+
+> A tool for displaying window and font properties in an X server.
+> More information: <https://manned.org/xprop>.
+
+- Display the name of the root window:
+
+`xprop -root {{WM_NAME}}`
+
+- Display the window manager hints for a window:
+
+`xprop -name {{window_name}} {{WM_HINTS}}`
+
+- Display the point size of a font:
+
+`xprop -font {{font_name}} {{POINT_SIZE}}`
```

Comment:
Can font names contain spaces? If so, this should be wrapped in double quotes.

Reply:
Obviously they can. (`Times New Roman`) I don't agree, there are many font names without spaces and users are smart, or do we have a hard rule on quoting everything that may contain spaces? (like file names 😉)

Reply:
There's no hard rule, but I would suggest it here as it aids clarity for the beginner. Your choice what to do

Reply:
Well, I don't think, it is a problem, because `xprop` is really not a program that any beginner is gonna use. But I don't have a huge problem with changing it, either.